tests/components/timer/test_init.py

```diff
@@ -92,7 +128,9 @@
     assert "0:00:10" == state_2.attributes.get(ATTR_DURATION)
 
     assert STATUS_IDLE == state_3.state
-    assert str(DEFAULT_DURATION) == state_3.attributes.get(CONF_DURATION)
+    assert str(cv.time_period(DEFAULT_DURATION)) == state_3.attributes.get(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4821917)